.github/workflows/ci.yml

```diff
@@ -48,6 +48,3 @@ jobs:
 
             - name: Build
               run: yarn build
-
-            - name: Test
```

Comment:
NOOOOOOOOOOOOOOOOOO
dont remove the entire pipeline xDDD
only remove the .test.tsx case, just like you did, but leave the pipeline alone lmao